domains/iosviet.json

```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "huutanphp",
+           "email": "",
+           "discord": "595368972594184235",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.UUB3u8JnQhweiOtS8ICf09sYk-WZL0haW_S50x1JgijmaiIBNCcX8hzE5xJVR2QXtiR_DfbseMXMtHVq5p_9SHf-qyiD8LKd22TKgorKH30PxP2Ej6Qw9LhbQKdI7EuAlBBGKyKm9gLWd8HYVeQlXMuya5X6AEKQS4WoH_GuX3itLnFUKf1hFZpEb5lJrdRKrvbTQgTXkLHsXggUi4Crcdfav4lmsA7DidmmfqpwXVy90TXdluqExc70JrtIY_9q0NDXwBe5boFOFOEYh7GW9MG0QHhor2Xibnd0pi5P5jAt9fUCGhUMMHKWTLBMjLumgswe8XEW4JXQGwmBBVMt_A.KpU_DlcxDclPoZwMO0H7rA.igZyUYSqQEyzpCfjGYAQivF9Jk8A6TzvA6sf2Ju7c4jP617xNtd7ZtTz2Wy9eYeSyF0ZDeEXn5RSWphywrSaIB6jhvzuEQ4xERnRW8ozdN0.xLh6AUpP7bDHQ7P8DwaZNw"
+        },
+    
+        "record": {
+            "A": ["3.106.238.127"]
```

Comment:
Just get a nginx landing page, can you send a preview of the site?

Reply:
<img width="1280" alt="Screen Shot 2024-09-22 at 04 53 55" src="https://github.com/user-attachments/assets/d926a7f3-1af3-407e-9f4d-b57c46e55325">

Reply:
Yeah that's just the default page, there's no content. Is there a site or not?

Reply:
![E3836C63-0E82-4097-9603-207E42C7BE6B](https://github.com/user-attachments/assets/1c435967-6c72-47c2-a54b-c9707cbecb98)